packages/wasm/crate/src/storage.rs

```diff
@@ -94,8 +94,10 @@ impl<T: Database> Storage<T> {
                 continue;
             }
 
-            if Some(record.note.asset_id()) != asset_id {
-                continue;
+            if let Some(id) = asset_id {
+                if record.note.asset_id() != id {
+                    continue;
+                }
```

Comment:
Bug fix 🐞 : The query should respect passing a `None`. Currently breaks delegator vote planning.